_releases/0.15.1.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Bitcoin Core 0.15.1
+id: en-release-0.15.1
+name: release-0.15.1
+permalink: /en/releases/0.15.1/
+excerpt: Bitcoin Core version 0.15.1 is now available
+
+## Use array so that sort order is [0.1.1, 0.10.0] rather than opposite
+release: [0, 15, 1, 0]
```

Comment:
The links are dead because of the extra ", 0" here.

Reply:
Huh weird, all the previous releases like 0.15.0 and 0.14.2 had that fourth zero there, so I assumed it was deliberate